src/services/company.js

```diff
@@ -15,6 +12,49 @@ class CompanyService {
                 ]
         });
     }
+
+    /**
+     *
+     * @param {*} limit - Number of documents to return
+     * @param {*} offset - where to start the query (pagination - how many documents to skip, NOT how many pages to skip)
+     *
+     * @returns {companies, totalDocCount}
+     */
+    async findAll(limit, offset) {
+
+        const totalDocCount = await Company.estimatedDocumentCount();
+
+        return {
+            totalDocCount,
+            companies:
+                [...(await Company.find({})
+                    .sort({ Name: "desc" })
```

Comment:
```suggestion
                    .sort({ name: "desc" })
```

Reply:
Why is descending order the default? If we're having a default, it should be ascending :thinking: